src/vs/workbench/contrib/chat/browser/actions/chatAccessibilityHelp.ts

```diff
@@ -112,6 +112,7 @@ export function getAccessibilityHelpText(type: 'panelChat' | 'inlineChat' | 'qui
 		content.push(localize('chatEditing.discardAllFiles', '- Undo All Edits{0}.', '<keybinding:chatEditing.discardAllFiles>'));
 		content.push(localize('chatEditing.openFileInDiff', '- Open File in Diff{0}.', '<keybinding:chatEditing.openFileInDiff>'));
 		content.push(localize('chatEditing.viewChanges', '- View Changes{0}.', '<keybinding:chatEditing.viewChanges>'));
+		content.push(localize('chatEditing.viewPreviousEdits', '- View Previous Edits{0}.', '<keybinding:chatEditing.viewPreviousEdits>'));
```

Comment:
Similar to the below, I would suggest using the actual label of commands rather than this separate string, it will be localized independantly and may be different in other languages, also it has to be kept in sync manually in english too.

Reply:
#258199